dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -37,25 +57,62 @@ public ObjectCreationNode(
         this.constructor = constructor;
         this.arguments = arguments;
         this.classbody = classbody;
+        this.receiver = receiver;
     }
 
+    /**
+     * Returns the constructor node
+     *
+     * @return the constructor node
+     */
+    @Pure
     public Node getConstructor() {
         return constructor;
     }
 
+    /**
+     * Returns the arguments
```

Comment:
```suggestion
     * Returns the explicit arguments to the object creation
```

Reply:
Should we add `explicit` or something else to highlight that these are only the explicit arguments, not any receiver expressions that might be added?

Reply:
Thanks! I am not sure whether we should return the receiver(if exists) now, lets discuss tmr.